docs/api/backends/embedding_sft.md

```
@@ -0,0 +1,106 @@
+# Embedding SFT Backend
+
+> Contrastive embedding fine-tuning. The [`SentenceTransformersBackend`](#) powers the [`embedding_sft()`](/api/functions/embedding_sft) algorithm.
```

markdownlint-cli2 (0.23.2)

[warning] No empty links

(MD042, no-empty-links)